js/preferences/Preferences.js

```diff
@@ -103,6 +126,33 @@ export function save_on_run_task_set(save_on_run_task) {
     return save();
 }
 
+/**
+ * Get the desired runner type.
+ * @returns {RunnerType}
+ */
+export function runner_get() {
+    return prefs.runtime_opts.runner;
+}
+
+/**
+ * The default runner type used globally.
+ * @returns {RunnerType}
+ */
+export function runner_project_get() {
```

Comment:
I'm not really sure why there's two of these

Reply:
I think I did the runner getter/setter functions for ProjectProperties first, then pasted and modified into Preferences but forgot to remove the `_project` functions that aren't needed for it